src/app/components/content-page/content-page.component.scss

```diff
@@ -1,51 +1,5 @@
-$navbar-color: #fafafa;
-$navbar-border-color: #bdbdbd;
-$default-border: 1px solid $navbar-border-color;
-
-header {
-  background: $navbar-color;
-  border-bottom: $default-border;
-}
-
-nav {
-  background: $navbar-color;
-}
-
 main {
   padding: 80px 0px;
   display: grid;
   grid-template-columns: auto auto auto;
-}
-
-.img-button {
-  background-color: rgba(0, 0, 0, 0);
-}
-
-.item-img {
-  object-fit: cover;
-}
-
-.bottom-bar {
-  background: $navbar-color;
-  border-top: $default-border;
-}
-
-input[type="file"] {
-  display: none;
-}
-
-.hand-cursor {
-  cursor: pointer;
-}
-
-@media screen and (max-width:400px) {
-  main {
-    grid-template-columns: auto auto;
-  }
-}
-
-@media screen and (max-width:300px) {
-  main {
-    grid-template-columns: auto;
-  }
-}
+}
```

Comment:
Why is this all moved to styles.css? Shouldn't this be moved to the home-feed component?

Reply:
In this task I didn't have a separated nav bar component. Both the Profile and the Home Feed used the styles that is why it is put in global styles. I will move them in the next task to the nav bar .scss file.